packages/client/src/query/aggregations/GroupByMapper.ts

```diff
@@ -27,4 +28,6 @@ export interface GroupByMapper {
   byte: NumericGroupByValue;
   double: NumericGroupByValue;
   integer: NumericGroupByValue;
+  timestamp: TimestampGroupByValue;
+  datetime: TimestampGroupByValue;
```

Comment:
Can we do this? Or does datetime need its own definition that doesn't let you do anything smaller than a day?

Reply:
Based on the backend code, date also seems to accept things smaller than day, but I'll double check with backend team